test/iron-overlay-behavior.html

```diff
@@ -1039,6 +1039,23 @@
 
       });
 
+      suite('animations', function() {
+
+        test('overlay animations correctly triggered', function(done) {
```

Comment:
> cool

I'm guessing that was sarcastic, but thanks for going the distance :beers: I really appreciate it.